linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -2558,6 +2668,27 @@ private[emitter] class FunctionEmitter(jsGen: JSGen) {
       genCallHelper("fround", arg)
     }
 
+    private def wrapBigInt32(tree: js.Tree)(implicit pos: Position): js.Tree =
+      wrapBigIntN(32, tree)
+
+    private def wrapBigInt64(tree: js.Tree)(implicit pos: Position): js.Tree =
+      wrapBigIntN(64, tree)
```

Comment:
Are these two helpers really worth it?

Reply:
Mostly to avoid typos such as 62 or 34. 32 and 64 have a pretty specific meaning. But I can't remove them if you prefer.

Reply:
It's fine with me to leave them.